content/docs/how_we_work/open_source_policy.md

```diff
@@ -0,0 +1,30 @@
+---
+# Page title as it appears in the navigation menu
+title: "Open Source Policy"
+# Adjust weight to reorder menu items (lower numbers appear first)
+weight: 2
+# Uncomment to hide nested pages in a collapsed menu
+# bookCollapseSection = true
+# Uncomment to hide this page from the navigation menu
+# bookHidden = false
+# Uncomment to exclude this page from the search
+# bookSearchExclude = true
+---
+
+# Open Source Policy
+
+REG has a policy of publishing all the work we do under an open source licence, unless there are compelling reasons to make an exception.
+This typically means publishing software we write under one of the [OSI approved licences](https://opensource.org/licenses), or in the case of non-code contributions one of the [Creative Commons licences](https://creativecommons.org/share-your-work/).
+Our default choices are
+
+- [BSD 3-clause licence](https://opensource.org/license/BSD-3-clause) for permissive source code licencing.
+- [AGPL](https://www.gnu.org/licenses/agpl-3.0.en.html) for copyleft code licencing.
+- [CC-BY](https://creativecommons.org/licenses/by/4.0) for permissive licensing of non-code content.
+- [CC-BY-SA](https://creativecommons.org/licenses/by-sa/4.0/deed.en) for copyleft licensing of non-code content.
+
+However, these are only defaults, and we [choose a licence](https://choosealicense.com/) based on the needs of each project.
+
+Open sourcing all our work is a strong default.
+We do make exceptions to it, but they need to be justified by some exceptional circumstance.
+The most typical case is working with a partner organisation and having to negotiate with them how to licence our joint work.
+In such cases we advocate for open sourcing the work to the greatest extent possible, but acknowledge that compromises may be necessary.
```

Comment:
```suggestion
In such cases we advocate for open sourcing the work to the greatest extent possible, but acknowledge that compromises may be necessary.

A further case would be where upstream or integrated software imposes its own licensing restrictions that prevent open sourcing.
We must always respect the licences of the code we use, whether open source or otherwise, but would also encourage the use of open source alternatives in such cases.
```

It might also make sense to highlight that we have to respect the licences of the code we use. Maybe this isn't the place for it, but it feels like an important point to mention somewhere.